src/__snapshots__/custom-sheet.test.js.snap

```diff
@@ -1,81 +1,81 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`1. general tests: media queries - {"fontSize":24,"margin":12,"@media (max-width: 641px)":{"fontSize":20,"margin":10}} 1`] = `
-.fs-24 {
+.glamor-0 {
   font-size: 24px;
 }
 
-.m-12 {
+.glamor-1 {
   margin: 12px;
 }
 
 @media (max-width: 641px) {
-  ._9gzisy {
+  .glamor-2 {
     font-size: 20px;
   }
 }
 
 @media (max-width: 641px) {
-  ._d2w6i1 {
+  .glamor-3 {
     margin: 10px;
   }
 }
 
 <div
-  className="fs-24 m-12 _9gzisy _d2w6i1"
+  className="glamor-0 glamor-1 glamor-2 glamor-3"
 />
 `;
 
 exports[`2. general tests: pseudo elements - {"& button":{"color":"green"}} 1`] = `
-.___button-c-green & button {
+.glamor-0 & button {
   color: green;
 }
 
 <div
-  className="___button-c-green"
+  className="glamor-0"
 />
 `;
 
 exports[`3. general tests: pseudo states - {":hover":{"backgroundColor":"blue"}} 1`] = `
-._hover-bc-blue:hover {
+.glamor-0:hover {
   background-color: blue;
 }
 
 <div
-  className="_hover-bc-blue"
+  className="glamor-0"
 />
 `;
 
 exports[`doesn't mess up stuff that does't have styles 1`] = `<div />`;
 
 exports[`enzyme.mount 1`] = `
-.p-4em {
+.glamor-3 {
   padding: 4em;
 }
 
-.b-papayawhip {
+.glamor-4 {
   background: papayawhip;
 }
 
-.fs-1p5em {
+.glamor-0 {
   font-size: 1.5em;
 }
 
-.text-align-center {
+.glamor-1 {
   text-align: center;
 }
 
-.c-palevioletred {
+.glamor-2 {
   color: palevioletred;
 }
 
 <Wrapper>
   <section
-    className="p-4em b-papayawhip"
+    className="glamor-3 glamor-4"
   >
     <Title>
       <h1
-        className="fs-1p5em text-align-center c-palevioletred"
```

Comment:
Ah, here's a problem I hadn't noticed before... We should only apply this to classes that are generated by `glamor`. So it must begin with `css-`. This is because:

1. You can provide glamor class names together with non-glamor class names
2. You can use this jest serializer with anything that implements a StyleSheet the way that glamor does (like cxs)